css/common.css

```diff
@@ -0,0 +1,415 @@
+@import url("font.css");
+.container {
+  margin: 0;
+  display: grid;
+  grid-template-columns: 16.375em 1fr;
+  grid-template-rows: 5.813em;
+  grid-template-areas:
+    "sidebar user-options-section"
+    "sidebar main";
+}
+body {
+  margin: 0;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Poppins", sans-serif;

```

Comment:
You have declared font-family for body twice with different values. It's better to use a single declaration and include fallback fonts to ensure consistency across browsers.

Reply:
I have made the required changes and included poppins as fallback font rather then declaring it again.